client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_second() {
```

Comment:
from https://blog.golang.org/examples
~~~
Example function names

Godoc uses a naming convention to associate an example function with a package-level identifier.

func ExampleFoo()     // documents the Foo function or type
func ExampleBar_Qux() // documents the Qux method of type Bar
func Example()        // documents the package as a whole
Following this convention, godoc displays the ExampleReverse example alongside the documentation for the Reverse function.

Multiple examples can be provided for a given identifier by using a suffix beginning with an underscore followed by a lowercase letter. Each of these examples documents the Reverse function:

func ExampleReverse()
func ExampleReverse_second()
func ExampleReverse_third()
~~~

I have no preference what its called as long as it passes the check.